### DISCUSSION TITLE
Lost my project after laptop crash

### DISCUSSION THREAD

**Discussion by TLegendar** [asreview/asreview#1659] [Troubleshooting (report errors in the "Issues" tracker)]
So well, my laptop crashed during screening with ASReview. I started my laptop back up and wanted to conitnue reviewing, but somehow my project was lost and isnt showing at the ASReview homepage. In the CMD line it looks like ASReview is trying to get my project but it wont succeed. This is also noticable in the homepage because it now gives me the option to try to refresh, but this wont succed either, it keeps on giving the refresh option after clicking it. Luckily I used Rayyan simultaneously with ASReview so I still know which articles I included and excluded. Does anybody know how to retrieve my project? I already updatet ASReview to the latest version.

**Comment by Rensvandeschoot** [comment]
ASReview [autosaves ](https://asreview.readthedocs.io/en/latest/screening.html#autosave) your project! Do you know where the `.asreview` folder is [stored](https://asreview.readthedocs.io/en/latest/generated/asreview.asreview_path.html) on your machine? 

Can you copy the text from the CMD so that we can help you to identify what is happening?

**Reply by TLegendar** [reply]
I know where the .asreview folder is stored. 

**CMD line:** 
127.0.0.1 - - [2024-01-23 14:04:43] "GET /api/projects/stats HTTP/1.1" 400 279 0.003036
ERROR:root:Expecting value: line 1 column 1 (char 0)

**Reply by Rensvandeschoot** [reply]
and the `.asreview` folder is not empty but contains your project?

**Reply by TLegendar** [reply]
I cant find a file in the .asreview folder that corresponds to the name I gave my project, if thats what you mean?

**Reply by Rensvandeschoot** [reply]
Are hou able to open the other projects in this folder via asreview?

**Reply by TLegendar** [reply]
No, because I only had one running/existing project. So there are no other projects that I can open. 

So im gussing my project data doesnt exist anymore? Because the project folder containing the autosave file doesnt exist anymore.

**Reply by J535D165** [reply]
Can you zip the content of your .asreview folder and send it to asreview@uu.nl? Can you also share with us what platform you are using? Windows/macOS?

**Reply by TLegendar** [reply]
I had no time to wait so I already deleted and reinstalled ASReview. After that I appointed all the articles that I already included and excluded to the relevant or irrelevant category (400 articles) and started screening again.

I will send you the details anyways, maybe there is still something to learn from it.

**Reply by IreneGN** [reply]
@Rensvandeschoot @J535D165  I have the same problem as this user. Luckily, I have not deleted / reinstalled ASReview yet and I have found the folder .asreview with my 6 projects (including the one that has "disappeared" from the dashboard). 
ASR Dashboard shows 5 projects only (the most recent one is gone after laptop crashed). ASR Dashboard also shows this message "Expecting value: line 1 column 1 (char 0)" and a "TRY TO REFRESH" button. When I click on it, nothing happens (apparently).
The error message I get from the CMD is:  ERROR:root:Expecting value: line 1 column 1 (char 0)
I use Windows. 
Is there any chance to recover my most recent project?
Thanks in advance.

**Reply by J535D165** [reply]
Sorry for the inconvenience, can you send the folder containing the failed project as a Zip to asreview@uu.nl?

**Reply by IreneGN** [reply]
@J535D165  @Rensvandeschoot Thanks for restoring the file in only a couple of hours!

I imported the project and I could continue labelling items without any problem. I was afraid that the most recently reviewed references would be missing... but I when I opened the project, it started excatly with the same reference as before the crash.

**Reply by Sujayadi79** [reply]
I also have the same problem. Can you also help me with this?

**Reply by Kaia010** [reply]
@J535D165 @Rensvandeschoot Hello, I have also encountered the same problem, I have spent a lot of time on this project, so I am very broken now, can you help me recover or tell me what I should do? If possible, thank you very much!

**Reply by J535D165** [reply]
Can you send me your project at jonathandebruinos@gmail.com? I will fix it.

**Reply by J535D165** [reply]
Everyone with a crashed project, send it to me and I will look into it. Don't worry. 

If you have a backup somewhere, please also send it to me (not essential).

**Reply by Kaia010** [reply]
@J535D165 Thank you very much for your reply! I'm so moved! Just now, I have solved this problem, and I found that the problem is in the project.json file, which is empty. Fortunately, I backed up this file once before, I replaced the faulty file with this backup file, and the missing project came back and worked correctly! Thanks again for your reply and help!

**Reply by J535D165** [reply]
Thanks, thats the solution! Your data is save. Happy screening!

Did this issue happen to you after a crash or power shortage as well? This helps us to figure out what causes this. Thanks

**Reply by Kaia010** [reply]
Yes, while I was running the project, my computer suddenly crashed, so I restarted the computer, and when I turned it on again, I found that only the project I was running had disappeared, but the other projects were still there.
